scripts/build.sh

```diff
@@ -35,19 +35,25 @@ function build_translation_archives {
   rm -f "$target_directory/*"
 
   for lang_dir in "$source_directory"/pages*; do
-    if [ -d "$lang_dir" ]; then
-      local lang=$(basename "$lang_dir")
-      local archive_name="tldr-$lang.zip"
+    # Skip symlinks (pages.en) and things that are not directories
+    if [ ! -d "$lang_dir" ] || [ -h "$lang_dir" ]; then
+      continue
+    fi
 
-      # Create the zip archive
+    local lang=$(basename "$lang_dir")
+    local archive_name="tldr-$lang.zip"
 
-      cd "$lang_dir"
-      zip -q -r "$target_directory/$archive_name" .
-      zip -q -j "$target_directory/$archive_name" "$source_directory/LICENSE.md"
+    # Create the zip archive
 
-      echo "Pages archive of $archive_name successfully created."
-    fi
+    cd "$lang_dir"
+    zip -q -r "$target_directory/$archive_name" .
+    zip -q -j "$target_directory/$archive_name" "$source_directory/LICENSE.md"
+
+    echo "Pages archive of $archive_name successfully created."
   done
+
+  cd "$target_directory"
+  ln -s tldr-pages.zip tldr-pages.en.zip
```

Comment:
```suggestion
  ln -s tldr.zip tldr-pages.en.zip
```
Shouldn't the old one to be symlinked be `tldr.zip`?

Reply:
`tldr.zip` is the full archive (all languages). `tldr-pages.zip` is only English.

Reply:
> `tldr.zip` is the full archive (all languages). `tldr-pages.zip` is only English.
> 
> `tldr-pages.en.zip` should also be only English.

Yeah, you are right. Sorry for the confusion. Tested the script locally and it works fine.